packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -309,7 +309,8 @@ sub-customer-error =
 sub-billing-update-success = Your billing information has been updated successfully
 sub-route-payment-modal-heading = Invalid billing information
 sub-route-payment-modal-copy = There seems to be an error with your Paypal account, we need you to take the necessary steps to resolve this payment issue.
```

Comment:
Request!  Let's replace 'Paypal' with `{ -brand-name-paypal }` on line 311 so we don't need to do it in another PR. 😁

Reply:
Good call; I copied that from before, but you're right, we're not using the placeable, and also it's not "PayPal" (two capital Ps)